smart-freight-center/sfc-backend
# feat: parse output after footprint data is retrieved

## Description
- Ensures that only fields in the data model would be returned to the user thus removing unknown fileds
- Corrects connector config bug
- Modifies endpoint in app to reflect
- Improves the initiate contract negotiation logic to not wait infinitely. 

### How to test it
- When you add unknown fields to your data source you would not see in the consumer response

Resolves https://github.com/smart-freight-center/sfc-backend/issues/182
